thirdparty/jquery-entwine/jquery.entwine.js

```diff
@@ -742,80 +744,19 @@ Sizzle is good for finding elements for a selector, but not so good for telling
 
 
 })(jQuery);
-;
-
-
-/* src/jquery.focusinout.js */
-
-(function($){	
-	
-	/**
-	 * Add focusin and focusout support to bind and live for browers other than IE. Designed to be usable in a delegated fashion (like $.live)
-	 * Copyright (c) 2007 Jörn Zaefferer
-	 */
-	if ($.support.focusinBubbles === undefined)  {
```

Comment:
`$.support` [is deprecated](https://api.jquery.com/jquery.support/), but thankfully since this was just to support IE we can remove it without repercussions.